### ISSUE TITLE
Realtime Statistics from Channels

### ISSUE THREAD

**Issue by hmelder** [TUM-Dev/VMP#13]
This may include:
- Bitrate
- Jitter
- Dropped Buffers
- QOS Events
- Number of Errors
- etc.

**Comment by joschahenningsen** [comment]
Can these be prometheus compatible metrics? :)

**Comment by hmelder** [comment]
Yep. I am currently reading the specification of the /metrics endpoint :)
